### PR TITLE
Add logaddexp

### PR DESCRIPTION
**Pull request by horta**

### CODE REVIEW

File: recipes/logaddexp/build.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+set -ex
+
+mkdir build && cd build
+
+cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX $SRC_DIR
```
Comment (hmaarrfk) [inline recipes/logaddexp/build.sh]:
```suggestion
cmake \
     -DCMAKE_BUILD_TYPE=Release \
     -DCMAKE_INSTALL_PREFIX:PATH=$PREFIX \
     ..
```

File: recipes/logaddexp/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja  # [win]
+  host:
+    - conda-build
```
Comment (hmaarrfk) [inline recipes/logaddexp/meta.yaml]: Is this necessary?

File: recipes/logaddexp/bld.bat [hunk 3]
```diff
@@ -0,0 +1,14 @@
+mkdir build && cd build
+
+cmake -GNinja ^
+    -DCMAKE_BUILD_TYPE=Release ^
+    -DCMAKE_INSTALL_PREFIX="%LIBRARY_PREFIX%" ^
+    :: -DCMAKE_INSTALL_LIBDIR="lib" ^
```
Comment (hmaarrfk) [inline recipes/logaddexp/bld.bat]: What are these colons for

File: recipes/logaddexp/meta.yaml (hunk 2 above)
Comment (horta) [reply recipes/logaddexp/meta.yaml]: No, thanks!

File: recipes/logaddexp/bld.bat (hunk 3 above)
Comment (horta) [reply recipes/logaddexp/bld.bat]: Fixed.

**Comment by horta** [comment]
Bump. Please, consider this package.

**Comment by hmaarrfk** [comment]
@conda-forge/staged-recipes please review. All looks good to me

**Comment by beckermr** [comment]
Maintainer username is wrong maybe? Should be a lowercase h though idk if GitHub is case sensitive.

File: recipes/logaddexp/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+  home: https://github.com/logaddexp/logaddexp
+  license: MIT
+  license_family: MIT
+  license_file: LICENSE
+  summary: 'C library that implements the logarithm of the sum of exponentiations'
+  dev_url: https://github.com/horta/logaddexp
+
+extra:
+  recipe-maintainers:
+    - Horta
```
Comment (hmaarrfk) [inline recipes/logaddexp/meta.yaml]:
```suggestion
    - horta
```

File: recipes/logaddexp/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,36 @@
...
+
+build:
+  number: 0
+  skip: true  # [win and vc<14]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - cmake
+    - ninja  # [win]
```
Comment (chrisburr) [inline recipes/logaddexp/meta.yaml]:
```suggestion
    - ninja  # [win]
    - make  # [unix]
```

**Pull request merged**